packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -143,6 +143,12 @@ export class FacetSearch {
     );
   }
 
+  private highlightSuggestion(suggestion: string) {
+    const search = this.props.controller.state.facetSearchQuery;
+    const regex = new RegExp(`(${search})`, 'ig');
```

Comment:
And apart from XSS escaping like Sami said, I think this would fail for any regex sensitive characters.

For example, if somebody types `|`, `[ ]`, `.`, `$`, etc. in the facet search box.